src/components/styles/careersection.css

```diff
@@ -0,0 +1,21 @@
+
+.tableC{
+    margin:15%;
+}
+.header{
+    font-family: 'Poppins';
+    font-style: normal;

```

Comment:
The default value of this property is already "normal", it is not necessary to write it